webextension/popup.html

```diff
@@ -60,6 +60,13 @@ <h3 class="onboarding-title">Always open sites in the containers you want.</h3>
     <a href="#" id="onboarding-done-button" class="onboarding-button">Done</a>
```

Comment:
This should be changed to:
```
 <a href="#" class="onboarding-button onboarding-almost-done-button">Next</a>
```

I'm also not seeing the panel-5 ever from my testing.